StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1516ElementsMustBeSeparatedByBlankLine.cs

```diff
@@ -216,8 +217,7 @@ private static void ReportIfThereIsNoBlankLine(SyntaxNodeAnalysisContext context
 
             var allTrivia = parent.DescendantTrivia(TextSpan.FromBounds(firstNode.Span.End, secondNode.Span.Start),
                 descendIntoTrivia: true,
-                descendIntoChildren: n => true)
-                .ToImmutableList();
+                descendIntoChildren: n => true);
```

Comment:
:question: (I know you weren't working on this but...) Isn't this the same as passing `descendIntoChildren: null`, or simply omitting the argument? If so, the `null` value will avoid some virtual calls when traversing the tree.